test/cfa/ntp_conf_test.rb

```diff
@@ -407,3 +419,21 @@ def ntp_conf(file)
     end
   end
 end
+
+describe CFA::NtpConf::TrustedkeyRecord do
+
+  let(:ntp) { ntp_conf(file) }
+
+  let(:file) { ntp_file("") }
+
+  it "adds proper type to conf when saving" do
+    ntp.records << subject
+    record = ntp.records.last
+    record.value = "1"
+    record.raw_options = ""
+    record.comment = "# path for keys file\n"
+    expect(record.value).to eq "1"
+
+    ntp.save
```

Comment:
Please, check that correct value is written to file. In this case, I guess:

`expect(file.content.lines).to include("trustedkey 1# path to keys file")`